components/motor/gpiostepper/gpiostepper.go

```diff
@@ -278,14 +278,14 @@ func (m *gpioStepper) doStep(ctx context.Context, forward bool) error {
 		return err
 	}
 	// stay high for half the delay
-	time.Sleep(m.stepperDelay / 2)
+	time.Sleep(m.stepperDelay / 2.0)
```

Comment:
let's make sure this never happens again,
does
```suggestion
	time.Sleep(m.stepperDelay * 0.5)
```
work?